src/aat/resources/features/F-026/F-026.feature

```diff
@@ -0,0 +1,54 @@
+@F-026
+Feature: F-026: Get case data with UI layout
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-078
+  Scenario: must return the list of cases and status code 200 for correct inputs
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
```

Comment:
this seems duplicated with the other scenarios, can't we pull it up to a common background section?

Reply:
I don't see why moving it to background would make it more difficult to read a scenario. Duplication is evil 99% of the times, especially in func tests. Better not to encourage this practice IMHO